### PR TITLE
Olevba compliant python3.5

### PR DESCRIPTION
**Pull request by sebdraven**
I don't change the regex logical. The code decode VBA macros with bytes and as soon as possible it decode in urtf-8 to match with regex. I modify also the export json because unicode doesn't exist in python 3.5. It's replaced by str and it's not the function to decode. I add the if the json_obj is bytes like hexstring.

### CODE REVIEW

**Comment by decalage2** [comment]
Hi @sebdraven, thanks a lot this is awesome! :-)

Did you test it with Python 2.7 too? I think some changes won't work as-is.

Also, please revert the changes to olefile2, that version is just there for backward compatibility of olefile with Python 2.5 and older (I will remove it anyway, since most of the code requires Python 2.6+).

**Comment by sebdraven** [comment]
In python 2.7 it's broken because it's not the same process to decode data in python 3 and python 2

I think it's better now to have two files. Olevba2 and olevba3

**Comment by sebdraven** [comment]
I revert olefile2 :)

**Comment by christian-intra2net** [comment]
Just stumbled across this, that's great! Must have been quite a bit of work, thanks for that. Have not tested it yet, but will surely do. 

From the first glance it looks great, just found 2 trivial things:
- some non-sense whitespace change in olefile2.py is left-over
- variable dencode in json2ascii can be removed, maybe even the complete function

**Comment by decalage2** [comment]
Hi @sebdraven, too bad I can't merge your PR as a whole, it will break Py2.x support. I will pick your changes one by one, and introduce them progressively to keep Py2+3 support. 

Or I need to merge it into a separate branch and work on it until it runs on Py2. (I'm not fluent in git yet ;-)

File: oletools/thirdparty/olefile/olefile.py [hunk 1]
```diff
@@ -1030,10 +1030,11 @@ def append_kids(self, child_sid):
         #[PL] this method was added to use simple recursion instead of a complex
         # algorithm.
         # if this is not a storage or a leaf of the tree, nothing to do:
+
         if child_sid == NOSTREAM:
             return
         # check if child SID is in the proper range:
-        if child_sid<0 or child_sid>=len(self.olefile.direntries):
```
Comment (decalage2) [inline oletools/thirdparty/olefile/olefile.py]: That change does not look right: why replace "<0" by "<=0"?

File: oletools/thirdparty/tablestream/tablestream.py [hunk 2]
```diff
@@ -233,24 +233,24 @@ def write_row(self, row, last=False, colors=None):
                 # encode to UTF8, avoiding errors
                 cell = cell.decode('utf-8', errors='replace')
             else:
-                cell = unicode(cell)
```
Comment (decalage2) [inline oletools/thirdparty/tablestream/tablestream.py]: not compatible with py2.7

**Pull request merged**